tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -143,6 +143,14 @@ def kill_all(self):
         self.marionette.switch_to_frame()
         self.marionette.execute_async_script("GaiaApps.killAll();")
 
+    def install_app(self, nameApp, urlApp):
```

Comment:
nameApp is not used, please remove it.
I would call the last argument manifestURL instead of urlApp.